src/lib/geoip/client.ts

```diff
@@ -126,40 +133,53 @@ export default class GeoipClient {
 			};
 		}
 
-		const maxmind = sources.find(s => s.provider === 'maxmind' && s.city === best.city);
-
-		if (maxmind?.asn && maxmind?.network) {
-			return {
-				asn: maxmind.asn,
-				network: maxmind.network,
-				normalizedNetwork: maxmind.normalizedNetwork,
-			};
+		for (const source of rankedSources) {
+			if (source.normalizedCity === best.normalizedCity && source?.asn && source?.network) {
+				return {
+					asn: source.asn,
+					network: source.network,
+					normalizedNetwork: source.normalizedNetwork,
+				};
```

Comment:
Not sure if this was discussed anywhere, but the previous logic intentionally always used ASN data from MaxMind because we considered it the best.

Reply:
I'm not saying this is wrong but the previous way was on purpose. We considered ASN data from other providers so bad that we decided it was better to not resolve the probe than to use them. @jimaek wdyt?

Reply:
I guess the ASN data from the winner does make sense, but I would push ipinfo to a much higher priority. Its the only one that is actually under active development and keeps adding improvements to their API and data.

Reply:
I can switch the priority along with the radius approximation logic, as ipinfo shows great results exactly when radius logic is applied.